### PR TITLE
Added complex_words.csv and added/updated functions for word cloud complexity.

### PR DESCRIPTION
**Pull request by BritVandy**
# Description

- added Tesseract instillation instructions for Mac in crop_cloud.py
- complex_words.csv was added to "data/crop-cloud"
- updated "data/README.md" to explain why complex_words.csv was added
- updated complexity_df function in wordcloud_functions.py to check complex_words.csv before assigning complexity scores
- two new functions were added to the wordcloud_functions.py: story_word_count function gives total story word count and get_top_complex_words gives a list of the number (choice) of top complex words

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Checklist

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

[Rubric](https://www.notion.so/lambdaschool/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

## Trello Cards
[As a parent I would like to see how many words my child averages within a story.](https://trello.com/c/0YfcYmim)
[Research appropriate metric for indicating word complexity.](https://trello.com/c/wfHY2Hsf)

[Loom Video](https://www.loom.com/share/52ff70f9f080408c85a25202eec9e837)

### CODE REVIEW

**Review by WhyseRabbit — COMMENTED** [review]
This list is well-thought-out, with much research obviously done to complete it. Great job!

File: data/crop-cloud/complex_words.csv [hunk 1]
```diff
@@ -0,0 +1,605 @@
+word,school level,complexity
```
Comment (peterger8y) [inline data/crop-cloud/complex_words.csv:1]: solid implementation of the csv!

**Review by peterger8y — COMMENTED** [review]
this should beef up the complexity algorithm nicely

**Review by peterger8y — APPROVED** [review]
This should beef up the complexity metric nicely!

**Review by afroman32 — APPROVED** [review]
This is a really cool addition. I like the way you implemented this into the existing complexity metrics.

**Pull request merged**